### DISCUSSION TITLE
shadowRoot with tailwindcss

### DISCUSSION THREAD

**Discussion by goodchengwang** [wxt-dev/wxt#829] [Q&A] [answered]
Using shadowRoot to create content scripts ui If the font size of the parent element's html is set to 12px, Tailwind's 1rem will be 12px instead of 16px.

**Comment by aklinker1** [comment, marked as answer]
Yes. I use this post-css plugin to convert from rem to px for my extensions: https://www.npmjs.com/package/postcss-pxtorem

**Reply by goodchengwang** [reply]
Thank you. That's a great idea.
